kpi-tracker/src/main.rs

```diff
@@ -0,0 +1,229 @@
+use std::collections::{BTreeMap, HashMap};
+use std::hash::Hash;
+
+use anyhow::Context;
+use chrono::{DateTime, Utc};
+use clap::Parser;
+use concordium_rust_sdk::{
+    id::types::AccountCredentialWithoutProofs,
+    smart_contracts::common::AccountAddress,
+    types::{
+        hashes::{BlockMarker, HashBytes},
+        AbsoluteBlockHeight,
+    },
+    v2::{self, AccountIdentifier, Client, Endpoint, FinalizedBlockInfo},
+};
+use futures::{self, StreamExt};
+
+#[derive(Debug, Parser)]
+struct Args {
+    /// The node used for querying
+    #[arg(long, default_value = "http://localhost:20001")]
```

Comment:
Add `help` attribute as well. Also please be explicit about naming, so `long = "node"`. Experience suggests that is more stable. We don't want renaming of an internal variable to change an external API of the service.

In the help specify that this is V2 API.